src/test/java/api/support/builders/OverrideCheckOutByBarcodeRequestBuilder.java

```diff
@@ -16,21 +18,25 @@
   private final String servicePointId;
   private final DateTime dueDate;
   private final String comment;
+  private final OverrideBlocks overrideBlocks;
 
   public OverrideCheckOutByBarcodeRequestBuilder(
```

Comment:
Just a thought: Can this class be merged with or replace CheckOutByBarcodeRequestBuilder and take on the name "CheckOutByBarcodeRequestBuilder" since it has almost the same code as CheckOutByBarcodeRequestBuilder and also the override code in it to simplify things?

Reply:
Yes, OverrideCheckOutByBarcodeRequestBuilder.java will be removed in scope on another ticket. It was updated to fix broken existing tests